### PR TITLE
TileSet&TextureRegion: fix error when getting nonexistent tile

### PR DESCRIPTION
**Pull request by ghost**
Fixes these:
image

### CODE REVIEW

File: editor/plugins/texture_region_editor_plugin.cpp [hunk 1]
```diff
@@ -57,8 +57,11 @@ void TextureRegionEditor::_region_draw() {
 		base_tex = obj_styleBox->get_texture();
 	else if (atlas_tex.is_valid())
 		base_tex = atlas_tex->get_atlas();
-	else if (tile_set.is_valid() && selected_tile != -1)
-		base_tex = tile_set->tile_get_texture(selected_tile);
+	else if (tile_set.is_valid() && selected_tile != -1) {
+		if (tile_set->has_tile(selected_tile))
```
Comment (akien-mga) [inline editor/plugins/texture_region_editor_plugin.cpp]: I guess you could put this additional condition on the main one after `&&` ?

Comment (Noshyaar) [reply editor/plugins/texture_region_editor_plugin.cpp]: Fixed :)

**Comment by hpvb** [comment]
Note to self: This currently isn't needed for 3.0.3 but if we merge the other related work into 3.0.x then this should come in also

**Comment by akien-mga** [comment]
Now that Godot 3.1 has been released, we only plan to cherry-pick critical bug fixes to the 3.0 branch, unless there is very strong support in favor of a specific non-critical issue. Removing cherrypick label for 3.0.

**Pull request merged**